src/index.js

```diff
@@ -13,7 +14,7 @@ app.use(express.json());
 app.use(express.urlencoded({extended: true}));
 
 // Routes
-// app.use("/example", exampleRouter)
+app.post("/register", register);
```

Comment:
I would create a user router file (registering is the same as creating a user) in the controller folder for better organization otherwise the index.js file is going to get overfilled with paths. There is an example in examples/week2/CRUD demo of the training repo on how to use express.router().